rest_framework_json_api/templates/rest_framework_json_api/includes.html

```diff
@@ -0,0 +1,38 @@
+<div class="modal fade" id="includesModal" tabindex="-1" role="dialog" aria-labelledby="includes" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title">JSON:API includes</h4>
```

Comment:
we won't provide trans files just now as this is the first time we actually need it. It would be good anyhow to use `{% trans` as used above in `api.html` to be ready if we will in the future.